app/src/main/java/echoserver/client/ClientSocketWrapper.java

```diff
@@ -48,6 +52,16 @@ public String receiveResponse() throws IOException {
     return message;
   }
 
+  public boolean quit(String message) {
```

Comment:
Nit: Looks like the preferred naming convention for `boolean`s in Java is prepending a verb (ie. `hasCats`)